packages/augur-ui/src/modules/modal/components/modal-universe-selector.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import { DefaultButtonProps } from 'modules/common/buttons';
+import {
+  Title,
+  Description,
+  ButtonsRow,
+} from 'modules/modal/common';
+import { Universe } from 'modules/types';
+import UniverseCard from 'modules/universe-cards/containers/universe-card';
+
+import Styles from 'modules/modal/modal.styles.less';
+import { UniverseDetails } from '@augurproject/sdk/src/state/getter/Universe';
+
+interface ModalUniverseSelectorProps {
+  title: string;
+  description: string;
+  universeDetails: UniverseDetails,
+  buttons: DefaultButtonProps[];
+  closeAction: Function;
+}
+
+export const ModalUniverseSelector = ({
+  title,
+  description,
+  universeDetails,
+  buttons,
+  closeAction
+}: ModalUniverseSelectorProps) => {
+  return(
```

Comment:
might need a call to an action to populate universes that calls `loadUniverseDetails` for the parent then for the children.

this modal can be made up of multiple UniveseCards. 

prob show the parent (current) 
then `map` over the children.

Reply:
Yeah, I have some changes for that for displaying multiple universe cards. Was going to do that as a separate PR, but maybe I'll just add it to this one.